### ISSUE TITLE
Doesn't seem to run on Sierra

### ISSUE THREAD

**Issue by bluestarstudios** [czarny/Brisync#12]
Hi czarny!

I've installed the app, but when I try to launch it, it never opens. I checked activity monitor and can't see it anywhere. Also checked if it was being blocked by the security in System Preferences –> Security, but no. I tried running batchmod on it to reassign permissions, but that didnt' help either. Any ideas?

Also, should I have ddcctl installed for this?

Thanks!
David

**Comment by czarny** [comment]
Hi, it is hard to say what happens, ddcctl is not needed for sure. Please try find something in the system console. Let me know if you will get any error message or crash report in the logs.

**Comment by bluestarstudios** [comment]
Cześć Krzysztof!

Thanks for the info! Turns out a simple reboot and a SMC reset did the trick. The app works like a charm now. Thanks for the awesome work!
